### ISSUE TITLE
HTTPTunnel can use HTTP tunneling via GET method ?

### ISSUE THREAD

**Issue by ingnelson** [lesterpotter/HTTPTunnel#1]
Hello.

HTTPTunnel can use HTTP tunneling via GET method ?

becuase my firewall blocks the HTTP request connections that beging with CONNECT method.

and i want to create a http tunnel using GET method, using Android device as a client and install HTTPTunnel on my VPS , so this way i can use the tunnel as a VPN.

I will wait your answer.

**Comment by lesterpotter** [comment]
Yes, this HTTPTunnel uses GET, PUT, POST and DELETE to implement the tunnel.  It does not use CONNECT.

**Comment by ingnelson** [comment]
@lesterpotter Could be posible you create a Android App to use it as a client and connect to the server (VPS ) and this way create the tunnel , please ?

because in many countries it is the only way to connect to internet . This can help to many people.

Please .

I will wait your answer.

**Comment by lesterpotter** [comment]
I would think so.  The client code would have to be translated to java, and converted to android socket implementation (very different), wrapped in an app.  But it's just a tunnel, not a proxy, Your browser needs a local proxy that can talk to the tunnel, and a remote proxy that's listening to the other end of the tunnel.

**Comment by ingnelson** [comment]
@lesterpotter how about to use Termux app on Android ?
Termux is Linux terminal for Android .

Do you have a telegram ?
